packages/components/src/__next__/Button/_docs/Button--migration-guide.mdx

```diff
@@ -0,0 +1,81 @@
+import { Canvas, Meta, Controls, Story } from '@storybook/blocks'
+import { ResourceLinks, KAIOInstallation } from '~storybook/components'
+
+<Meta title="Components/Button/Button (next)/Migration Guide" />
```

Comment:
I've been putting these at the top level 

```suggestion
<Meta title="Components/Button/Migration Guide" />
```

Reply:
Ah yep, thanks for the reminder! :)